packages/devtools_app/test/debugger/program_explorer_test.dart

```diff
@@ -78,6 +78,8 @@ void main() {
       when(fakeServiceManager.connectedApp!.isDartWebAppNow).thenReturn(false);
 
       final mockScriptManager = MockScriptManager();
+      //`then` is used
```

Comment:
you could probably just make setUpAll's closure async:
`    setUpAll(() async {`